src/typography/src/Link.js

```diff
@@ -56,6 +62,7 @@ class Link extends PureComponent {
         className={cx(className, themedClassName)}
         textDecoration="underline"
         color={null}
+        size={size}
```

Comment:
You can already pass the size prop to a link via `...props` I believe. 

What is the intended goal of this change?